src/config.rs

```diff
@@ -35,16 +35,17 @@ pub enum RawColor {
 }
 
 impl From<RawColor> for Color {
+    #[must_use]
```

Comment:
Hm, I don't see a big reason for these must use annotations. Also, the docs say:

> Not bad at all, this lint just shows places where you could add the attribute.

I'd remove all `#[must_use]` annotations and add `#[allow(clippy::must_use_candidate)]` to `src/main.rs` instead. (There are some more cases further down.)

Reply:
Ok, makes sense for me. In fact, after updating to 1.41, clippy no longer lints those.